packages/site/src/utils/snap.ts

```diff
@@ -67,4 +68,53 @@ export const sendHello = async () => {
   });
 };
 
+/**
+ * Toggle paymaster usage.
+ */
+
+export const togglePaymasterUsage = async () => {
+  await window.ethereum.request({
+    method: 'wallet_invokeSnap',
+    params: {
+      snapId: defaultSnapOrigin,
+      request: { method: 'snap.internal.togglePaymasterUsage' },
```

Comment:
I don't see this method listed in the `InternalMethod` enum [here](https://github.com/MetaMask/snap-account-abstraction-keyring/blob/fix/chain-config-setup/packages/snap/src/permissions.ts#L3)
I also don't see any logic for [handling this request](https://github.com/MetaMask/snap-account-abstraction-keyring/blob/fix/chain-config-setup/packages/snap/src/index.ts#L58-L69) in the keyring

Reply:
Yes this is in a subsequent pr to add deployment of the paymaster from the UI